src/features/bumpkins/components/BumpkinEquip.tsx

```diff
@@ -136,41 +136,7 @@ export const BumpkinEquip: React.FC<Props> = ({ equipment, onEquip }) => {
   const isMissingShirt = !equipped.shirt && !equipped.dress;
   const isMissingPants = !equipped.pants && !equipped.dress;
 
-  const warn =
-    isMissingHair ||
-    isMissingBody ||
-    isMissingShoes ||
-    isMissingShirt ||
-    isMissingBackground ||
-    isMissingPants;
-
   const { t } = useAppTranslation();
```

Comment:
We should continue to support the existing rules for bumpkin equip. 

Instead if a player sells something that is one of the required bumpkin items. it will fallback to the default that we set
```suggestion
  const warn =
    isMissingHair ||
    isMissingBody ||
    isMissingShoes ||
    isMissingShirt ||
    isMissingBackground ||
    isMissingPants;

  const { t } = useAppTranslation();
  const warning = () => {
    if (isMissingHair) {
      return t("equip.missingHair");
    }

    if (isMissingBody) {
      return t("equip.missingBody");
    }

    if (isMissingShoes) {
      return t("equip.missingShoes");
    }

    if (isMissingShirt) {
      return t("equip.missingShirt");
    }

    if (isMissingPants) {
      return t("equip.missingPants");
    }

    if (isMissingBackground) {
      return t("equip.missingBackground");
    }
    return "";
  };
```